app/Http/Controllers/Group/AboutController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers\Group;
+
+use App\Http\Controllers\Controller;
+use App\Models\Group;
+use Illuminate\Http\Request;
+
+class AboutController extends Controller
+{
+    public function show(Group $group)
+    {
+        return view('groups.about', compact('group'));
```

Comment:
So, views should also have the method at the end? In this case it would be `groups.about.show`?

In that case, they would only implement: `index`, `show`, `create`, `edit`?

These wouldn't be used for views: `store`, `update`, `delete`?

Did I get this right?

Reply:
Do I also need `dashboard.show` and `login.show`?

Reply:
I think the point on this one in particular - discounting special cases like `dashboard/login/logout/home/welcome`, is that either this is the `groups.show` page and the other thing is the `groups.bills.index` page, or this is the `groups.members.index` page and the other is the `groups.show` page. But either way one or both of your routes is miscategorized.

Reply:
I don't understand what you are saying here.

Reply:
In a nutshell, `about` is not a thing in a CRUD app. (Which, pretty much all non-pamphlet Laravel apps are)

So how can you reorganize the routes and controller names to delete that word and use CRUD terms and organization instead?

It's a page that shows properties of a single group. It's constituent members, name, etc etc. Generally, that's a `show` page. 

Your other current `.show` page is really showing a list of things. What are the things it's showing a list of? That's probably a `groups.<that-model-name-plural>.index` route.

As I mentioned initially, your route organization does not make any sense and the way you're trying to organize is leading you into really unhealthy patterns like returning 3 views conditionally from a `show()` method and putting CRUD route keys into the `session()`. It's all connected.

If you can figure out this one piece, it may untangle other aspects for you to be able to reorganize into a sane overall principle and eliminate the unhealthy aspects of the current approach.

Reply:
Ok, I changed `groups.about.show` to `groups.members.index`.